src/test/extensions/resolveSource.spec.ts

```diff
@@ -1,45 +0,0 @@
-import { expect } from "chai";
```

Comment:
Delete this file

Reply:
It is - that's why there's no diff on the other side :)